pkg/plugins/layout_test.go

```diff
@@ -161,6 +162,7 @@ var CmdsAddAndExpandPart []console.CmdMock = []console.CmdMock{
 /some/device1 part`},
 	{Cmd: "mkfs.ext2 -L MYLABEL /some/device1"},
 	{Cmd: "growpart /some/device 1"},
+	{Cmd: "lsblk -ltnpo name /some/device", Output: "/some/device1\n/some/device4"},
```

Comment:
I think this is enough of a test. If it's not there the test fails. If I list non-matching partitions it also fails. Should be good to merge.